src/Aquarius.ONE.ClientSDK/Common/Configuration/Graph/EnumRelativeDateRange.cs

```diff
@@ -1,15 +0,0 @@
-﻿namespace ONE.Common.Configuration.Graph
-{
-    public enum EnumRelativeDateRange

```

Comment:
There is a similar sounding enum over in: 

https://github.com/AquaticInformatics/aquarius-one-sdk/blob/9440b70c819401a8cc2bf1249a7b57bd941d5cb1/src/Aquarius.ONE.ClientSDK/Enterprise/Report/EnumRelativeDateRange.cs

That one has less options than the replaced enum, so might be intentional that it is kept.

Reply:
Removed